### PR TITLE
Added v1.0/applicationTemplate

### PR DESCRIPTION
**Pull request by sureshja**
This API exists in /beta already and we are GA'ing it as is now.

### CODE REVIEW

File: api-reference/v1.0/api/applicationtemplate-get.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+title: 'Get applicationTemplate'
```
Comment (Lauragra) [inline api-reference/v1.0/api/applicationtemplate-get.md:2]: I don't know if it matters, but generally the YAML metadata uses quotes for the values, not apostrophes. Not sure why these are like this.

File: api-reference/v1.0/api/applicationtemplate-get.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+If successful, this method returns a `200 OK` response code and the requested [applicationTemplate](../resources/applicationtemplate.md) object in the response body.
+
+## Examples
+
+### Request
+
+The following is an example of the request.
+
+# [HTTP](#tab/http)
```
Comment (Lauragra) [inline api-reference/v1.0/api/applicationtemplate-get.md:62]: Global: Delete the HTTP H1s from the topics.

File: api-reference/v1.0/api/applicationtemplate-get.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+```msgraph-interactive
+GET https://graph.microsoft.com/v1.0/applicationTemplates/{id}
+```
+
+### Response
+
+The following is an example of the response.
+
+> [!NOTE]
```
Comment (Lauragra) [inline api-reference/v1.0/api/applicationtemplate-get.md:77]: Global: We don't use this Note format for this boilerplate note. We just use the old ">" note format.

File: api-reference/v1.0/api/applicationtemplate-get.md (hunk 1 above)
Comment (sureshja) [reply api-reference/v1.0/api/applicationtemplate-get.md:2]: I see this everywhere including application resource docs. Pl let me know if you want me to update all these files.

File: api-reference/v1.0/api/applicationtemplate-get.md (hunk 3 above)
Comment (sureshja) [reply api-reference/v1.0/api/applicationtemplate-get.md:77]: Done

File: api-reference/v1.0/api/applicationtemplate-get.md (hunk 2 above)
Comment (sureshja) [reply api-reference/v1.0/api/applicationtemplate-get.md:62]: Done.

File: concepts/changelog.md [hunk 4]
```diff
@@ -11,6 +11,14 @@ This changelog covers specific API-level changes in Microsoft Graph v1.0 and bet
 
 For a summary of the value of these API changes, as well as recent tools, components, guidance, and tutorial additions, see [What's new in Microsoft Graph](whats-new-overview.md).
 
+## March 2021
```
Comment (Lauragra) [inline concepts/changelog.md:14]: The changelog.md file is deprecated; you'll want to add an entry to the DirectoryServices.json file in the changelog folder for this change.

Comment (sureshja) [reply concepts/changelog.md:14]: Done.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**